src/CssToInlineStyles.php

```diff
@@ -101,7 +101,7 @@ public function getInlineStyles(\DOMElement $element)
     protected function createDomDocumentFromHtml($html)
     {
         $html = trim($html);
-        if (strstr('<?xml', $html) !== 0) {
+        if (strpos('<?xml', $html) !== 0) {
```

Comment:
Funny thing is that this bug has no effect since adding xml header multiple times is ignored by DomDocument therefore is difficult to write failing test in this PR :see_no_evil: